data_management/management/commands/add_example_data.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand
+from django.db.utils import OperationalError
+
+from data_management.management.commands._example_data import init_db
+from data_management.models import Object
+
+
+class Command(BaseCommand):
+
+    help = "Add example data to the registry"
+
+    def add_arguments(self, parser):
+        """
+        Define the options.
+
+        """
+        parser.add_argument(
+            "--force",
+            action="store_true",
+            help="Add the example data even if the database is not empty",
+        )
+
+    def handle(self, **options):
+
+        force = options["force"]
+
+        try:
+            user = get_user_model().objects.all()
+            group = Group.objects.all()
+            objects = Object.objects.all()
+
+            if (len(user) > 0 or len(group) > 0 or len(objects) > 0) and not force:
```

Comment:
Admin user is automatically created so `len(user)` will always be greater than 0